packages/authentication/test/acceptance/basic-auth.ts

```diff
@@ -127,24 +127,24 @@ describe('Basic Authentication', () => {
     class MySequence implements SequenceHandler {
       constructor(
         @inject('findRoute') protected findRoute: FindRoute,
-        @inject('getFromContext') protected getFromContext: getFromContext,
+        @inject('getFromContext') protected getFromCntxt: getFromContext,
```

Comment:
`getFromContext` is a type imported from `internal-types.ts`. At the same time, it's a property name defined on `MySequence` class. I'll fix this issue by renaming the type to `GetFromContext`, which is consistent with other existing type names like `FindRoute`.